test/extended/operators/management_plane_operators.go

```diff
@@ -0,0 +1,420 @@
+			"InstallerPodContainerWaitingDegraded",
+			"InstallerPodNetworkingDegraded",
+			"InstallerPodPendingDegraded",
+			"KubeControllerManagerStaticResourcesDegraded",
+			"MissingStaticPodControllerDegraded",
+			"NodeControllerDegraded",
+			"NodeInstallerDegraded",
+			"NodeInstallerProgressing",
+			"ResourceSyncControllerDegraded",
+			"RevisionControllerDegraded",
+			"StaticPodsAvailable",
+			"StaticPodsDegraded",
+			"TargetConfigControllerDegraded",
+			"UnsupportedConfigOverridesUpgradeable",
+		},
+		{Group: "operator.openshift.io", Version: "v1", Resource: "kubestorageversionmigrators"}: {
+			"DefaultUpgradeable",
+			"KubeStorageVersionMigratorAvailable",
+			"KubeStorageVersionMigratorDegraded",
+			"KubeStorageVersionMigratorProgressing",
+			"KubeStorageVersionMigratorStaticResourcesDegraded",
+		},
+		{Group: "operator.openshift.io", Version: "v1", Resource: "networks"}: {
+			"Available",
+			"Degraded",
+			"ManagementStateDegraded",
+			"OVNKubernetes",
+			"Progressing",
+			"Upgradeable",
+		},
+		{Group: "operator.openshift.io", Version: "v1", Resource: "openshiftapiservers"}: {
+			"APIServerDeploymentAvailable",
+			"APIServerDeploymentDegraded",
+			"APIServerDeploymentProgressing",
+			"APIServerStaticResourcesDegraded",
+			"APIServerWorkloadDegraded",
+			"APIServicesAvailable",
+			"APIServicesDegraded",
+			"AuditPolicyDegraded",
+			"ConfigObservationDegraded",
+			"Encrypted",
+			"EncryptionKeyControllerDegraded",
+			"EncryptionMigrationControllerDegraded",
+			"EncryptionMigrationControllerProgressing",
+			"EncryptionPruneControllerDegraded",
+			"EncryptionStateControllerDegraded",
+			"OperatorConfigProgressing",
+			"ResourceSyncControllerDegraded",
+			"RevisionControllerDegraded",
+			"UnsupportedConfigOverridesUpgradeable",
+		},
+		{Group: "operator.openshift.io", Version: "v1", Resource: "openshiftcontrollermanagers"}: {
+			"Available",
+			"ConfigObservationDegraded",
+			"OpenshiftControllerManagerStaticResourcesDegraded",
+			"Progressing",
+			"ResourceSyncControllerDegraded",
+			"Upgradeable",
+			"WorkloadDegraded",
+		},
+		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecas"}: {
+			"Available",
+			"Degraded",
+			"Progressing",
+			"ResourceSyncControllerDegraded",
+			"Upgradeable",
+		},
+		{Group: "operator.openshift.io", Version: "v1", Resource: "storage"}: { // much isn't listed here and varies by platform
+			"ConfigObservationDegraded",
+			"CSIDriverStarterDegraded",
+			"DefaultStorageClassControllerAvailable",
+			"DefaultStorageClassControllerDegraded",
+			"DefaultStorageClassControllerProgressing",
+			"ManagementStateDegraded",
+			"VSphereProblemDetectorStarterDegraded",
+		},
+	}
+)
+
+var _ = g.Describe("[sig-arch][Early] Operators", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLIWithoutNamespace("management-plane-operators")
+
+	g.Describe("low level operators", func() {
+		g.It("should have at least the conditions we had in 4.17", func() {
```

Comment:
Would it be possible to tag this with `[apigroup:operator.openshift.io]` to enable skipping in MicroShift?

Reply:
Thanks!